Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/messaging/builder/impl/AbstractPRPAIN201305UV02Builder.java

```diff
@@ -136,4 +125,23 @@ protected void buildControlActProcess() throws PropertyAccessException {
     protected String getLocalHcid() throws PropertyAccessException {
         return PropertyAccessor.getInstance().getProperty(PROPERTY_FILE, LOCAL_HCID_KEY);
     }
+
+    private MCCIMT000100UV01Device getDevice(String hcid) {
+        MCCIMT000100UV01Device device = new MCCIMT000100UV01Device();
+        MCCIMT000100UV01Agent asAgent = new MCCIMT000100UV01Agent();
+        MCCIMT000100UV01Organization representedOrg = new MCCIMT000100UV01Organization();
+        II id = new II();
+        id.setRoot(hcid);
+        representedOrg.getId().add(id);
+
+        JAXBElement<MCCIMT000100UV01Organization> representedOrgJax
+            = new org.hl7.v3.ObjectFactory().createMCCIMT000100UV01AgentRepresentedOrganization(representedOrg);
```

Comment:
Is there a reason we're using the fully qualified class name, instead of importing the correct class?